include/translate-option.php

```diff
@@ -130,13 +130,16 @@ public function translate( $value ) {
 	 *
 	 * @param mixed      $values Either a string to translate or a list of strings to translate.
 	 * @param array|bool $key    Array of option keys to translate.
-	 * @return array|string Translated string(s)
+	 * @return array|string Translated string(s).
```

Comment:
This should avoid an ignored PHPStan error
```suggestion
	 * @return string|array|Traversable Translated string(s).
```

Reply:
But it creates a new one:
```
L120 - Parameter #2 $data of method PLL_Cache<array|string>::set() expects array|string, array|string|Traversable given.
```

Reply:
If we change things, that will be in a separate PR.